### PR TITLE
JS Library

### PR DESCRIPTION
**Pull request by Azeem838**
This project contains the following features:

1. Add a book
2. Mark "read" status
3. Delete a book
4. All written with JavaScript

Thank you.

### CODE REVIEW

**Review by Forison — CHANGES_REQUESTED** [review]
## Status:: Changes Required❌

# 🙋‍♂️Eric & Azeem,

#### You have done a great job on this one so far, however, there are a few changes we have to address to get this project approved🙇‍♂️.

* [x] Book pages can not be negative integers or zero, currently, the pages number field accepts negative values or zero. Let us address this issue.
* [x] Let us remove commented code from our index.html file. Kindly check line 11 of the ```index.html```
- Kindly remove this line ```<!-- <link rel="stylesheet" href="style.css"> -->```
* [x] The application has a addBookToLibrary function([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/book_array_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/library))
- Know this may sound a bit weird since ```addBook``` tends to do what ```addBookToLibrary``` seeks to do. But whilst working in a team when you are given a specific requirement as we have in the case of ```addBookToLibrary``` It is best to stick to that name given in the docs. 
* [x] It has a `render` function ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/render_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/library))
- Let us change the ```renderOne``` to ```render``` as stated in the requirement. 
* [x] Keep the application logic separated from DOM manipulation tasks ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from Pathwright).
- Let us remodel ```addBookToLibrary```, let us make it obey the single responsibility rule, thus making it only responsible for taking user input and add the book to the library and localStorage on click submit.

## That is it, Kindly make the changes and submit a re-review🤗.

#### Happy coding👨‍💻.

Addo Forison

File: scripts/library.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  const titleVal = title.value;
+  const numPagesVal = numPages.value;
+  const read = boxVal();
+
+  const book = new Book(authorVal, titleVal, numPagesVal, read);
+
+  myLibrary.push(book);
+
+  localStorage.setItem('myLibrary', JSON.stringify(book));
+
```
Comment (Torres-ssf) [inline scripts/library.js:96]: **OPTIONAL**
Your app successfully save the data in the local storage but does not load the saved data before the applications starts. You can also add this feature.

**Review by Torres-ssf — APPROVED** [review]
### Project Approved :+1: 
Hi team :hand: 
Nice job implementing the requested changes. You repository now match all requirements :clap: :tada: 
Good luck with the next one :smile: 

Happy Coding :computer: 
**Sergio Torres**

File: scripts/library.js (hunk 1 above)
Comment (EricMbouwe) [reply scripts/library.js:96]: Thanks for the review. We will add that feature for sure.

**Pull request merged**